### ISSUE TITLE
[bug] no mouse pointer captured?

### ISSUE THREAD

**Issue by dimitrieh** [cytopia/ffscreencast#8]
As noted in #5 I think ffscreencast is not capturing the mouse.. it this correct?

see following screencast:

![screencast](https://cloud.githubusercontent.com/assets/429659/14770168/1fc276cc-0a6b-11e6-9090-a7cd7979cf91.gif)

**Comment by cytopia** [comment]
On OSX, the default is to not capture the mouse.

You can add the option yourself to make the mouse visible:
```shell
ffscreencast --sargs="-capture_cursor 1"
```

If you also want to *see* when you actually click the mouse, do it like this:
```shell
ffscreencast --sargs="-capture_cursor 1 -capture_mouse_clicks 1"
```

I will also add this to the FAQ section. Thanks for reporting.

**Comment by dimitrieh** [comment]
Got ya! Handy to know

Op do 19 mei 2016 22:44 schreef cytopia <notifications@github.com>:

> Closed #8 <https://github.com/cytopia/ffscreencast/issues/8>.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/cytopia/ffscreencast/issues/8#event-666348480>
>
-- 
--

*Dimitrie Hoekstra, UI & Interaction Designer*
dhesign.com  | dimitriehoekstra@gmail.com | LinkedIn
<https://www.linkedin.com/in/dimitrieh> | Twitter
<https://twitter.com/strages33/>
